README.md

```diff
@@ -58,3 +58,11 @@ Okay, so we have all the ingredients laid out here
 *Note: do not mix formats in a single file, but they can be mixed with a folder*
 
 More information on the YouTube docs for [advanced transcibing](https://support.google.com/youtube/answer/2734698#zippy=%2Cbasic-file-formats%2Csubrip-srt-example%2Cadvanced-file-formats%2Cbroadcast-file-formats-tv-and-movies%2Csubviewer-sbv-example).
+
+To do this you would all be using the [youtube studio](https://studio.youtube.com/) having access to :
+ - Editing captions
+ - Adding languages
+ - Translating languages
+ and many more!
+ 
+ using the subtitle tab located at the left of personal youtube studio channel using your normal youtube channel as the authentication to accessing all this.
```

Comment:
thank you, but I think people will not have access to this apart from the admin of the YouTube account? I understand YouTube removed the ability for the community to do this. That is why we have a repo now, so an admin can download and add to this repo the English autogenerated one, then from there people can create the translations version.

Reply:
OK, Eddie